packages/components/inputs/radio-input/src/radio-option.styles.ts

```diff
@@ -18,6 +24,15 @@ const LabelTextWrapper = styled.div<TStylesProps>`
       : customProperties.fontColorForInput};
 `;
 
+const RadioOptionsWrapper = styled.div<TStylesProps>`
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: auto;
+  grid-template-areas:
+    'radio label'
+    '. content';
```

Comment:
I'm not sure it makes sense to use named areas here. For instance, if the same component is rendered multiple times in a page, wouldn't there be conflicts with the names?

I believe it should be possible to achieve the same with only `grid-template-columns` and `grid-template-rows`.